public/js/main.js

```diff
@@ -0,0 +1,209 @@
+/*
+ *	Heart of the Valley
+ *		main.js
+ */
+
+(async function() {
+  let res;
+  let art;
+
+  try {
+    res = await fetch('http://localhost:3000/api/artworks');
```

Comment:
do we want to use await, might be too new. Await over promises?

Reply:
@JMStudiosJoe Good call, looks like IE 11 doesn't even support the native `fetch` function https://caniuse.com/#search=fetch, I'll update that.